### PR TITLE
[localStorage] update localStorage api

### PR DESCRIPTION
**Pull request by spalger**
Tried taking a look at #55 

In the process I updated the `localStorage` usage to use `getItem`/`setItem`/`removeItem` and things seem better... Maybe it's just because I'm running things locally?

### CODE REVIEW

**Comment by ProLoser** [comment]
Not working, it tends to fix it when you restart the plugin (but I don't tell people that cuz that's stupid)

File: background.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ chrome.runtime.onMessage.addListener(function (message) {
 chrome.omnibox.onInputChanged.addListener(omni.suggest.bind(omni));
 
 chrome.omnibox.onInputStarted.addListener(function () {
-    if (_.isUndefined(localStorage.setup)) {
+    if (_.isUndefined(localStorage.getItem('setup'))) {
```
Comment (ProLoser) [inline background.js:30]: I actually think it's the call to `_.isUndefined()` because the value is null

**Comment by ProLoser** [comment]
@spalger can you see if this fixes it? https://github.com/ProLoser/Github-Omnibox/commit/03033a0b6382fb1ae793e1c0d87c933e6adc6601

**Comment by spalger** [comment]
LGTM

File: background.js (hunk 1 above)
Comment (spalger) [inline background.js:30]: hooray for `== null`

Comment (ProLoser) [inline background.js:30]: We were both wrong. For some reason i wasn't setting the `setup` flag after you made a decision.

**Pull request closed without merging**